packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -186,6 +188,34 @@ export class TablePageObject<T extends TableRecord> {
         return anchor as Anchor;
     }
 
+    public getRenderedCellIcon(
```

Comment:
I have a bit of a reservation about both this function and the equivalent function for the spinner below. They seem a bit too coupled to the mapping-icon column with their assumption that the icon/spinner will be the first child element. For example, `getRenderedCellAnchor` directly above doesn't do that.

I think the spinner version could look exactly like the anchor implementation since you know the element's tag. The icon version seems a bit trickier. I don't have a great alternative to the implementation you have off the top of my head, but perhaps if we can't come up with a more flexible selector, the function name should somehow indicate that it is specific to the `mapping-icon` column and isn't designed to be flexible for use with any column that could possibly contain an icon...just a thought, don't know if it's really the best solution here.

Reply:
We don't have to worry about the spinner function, because I removed it. I can't think of a more general way to get our icons (except to set a certain class on every Nimble icon), so I'm renaming the function.